Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs

```diff
@@ -75,6 +75,32 @@ public void Test_Query_NotFoundFilter()
         Assert.IsTrue(result.Data.Count == 0);
```

Comment:
https://github.com/FrendsPlatform/Frends.MongoDB/blob/b2a13e247113ab074d0d626c3450cc7d77b9e9f9/Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs#L48
No need to check if equal. You can check the value.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/b2a13e247113ab074d0d626c3450cc7d77b9e9f9/Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs#L49
Please use `AreEqual` instead of `IsTrue`.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/b2a13e247113ab074d0d626c3450cc7d77b9e9f9/Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs#L61
No need to check if equal. You can just assert the value.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/b2a13e247113ab074d0d626c3450cc7d77b9e9f9/Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs#L62
Please use `AreEqual` instead of `IsTrue`.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/b2a13e247113ab074d0d626c3450cc7d77b9e9f9/Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs#L74
No need to check if equal. You can just assert the value.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/b2a13e247113ab074d0d626c3450cc7d77b9e9f9/Frends.MongoDB.Query/Frends.MongoDB.Query.Tests/UnitTests.cs#L75
Please use `AreEqual` instead of `IsTrue`.